mne/viz/epochs.py

```diff
@@ -69,6 +71,10 @@ def plot_epochs_image(epochs, picks=None, sigma=0., vmin=None,
         Figure instance to draw the image to. Figure must contain two axes for
         drawing the single trials and evoked responses. If None a new figure is
         created. Defaults to None.
+    overlay_times : array-like, shape (n_epochs,) | None
+        If not None the parameter is interpreted as time instants in seconds
```

Comment:
Did anybody check how it looks like with unordered overlays? E.g. RTs, but not sorting by RT.